tests/Fixtures/JunitFormat/features/abort_on_php_error.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Abort on PHP error
+  In order to see correct information
+  As a features developer
+  I want to see the right output in JUnit when I abort due to a PHP error
+
+  Background:
+    Given I have entered 10
+
+  Scenario: Failed
+    When I add 4
+    Then I must have 14
```

Comment:
Obviously in real life the point is that any step could cause an error, but in the tests I think the indirection "I must have 14" => trigger an error again makes this harder to follow especially now that the steps and features are separated from the feature file.

Maybe we should just be explicit: `Then I trigger a PHP error`?

This might also again help with combining step definitions into a smaller number of contexts, to make it easier to follow what's going on?